### ISSUE TITLE
Support Write-Through Mode

### ISSUE THREAD

**Issue by jyore** [jyore/dynamo-cache#1]

**Comment by jyore** [comment]
- Will probably have to add a couple options to the `put` api to select the correct hash/sort keys

**Comment by Nedvedyy** [comment]
Jyore, any plan to implement it?

**Comment by jyore** [comment]
@Nedvedyy no, I think this project is pretty much dead at this point. There is really no need for it anymore, however, as AWS has added their own caching layer for Dynamo, which is super simple to use and likely better than this solution would have been. I should make a note of this on the project's README.

https://aws.amazon.com/dynamodb/dax/
